index.html

```diff
@@ -1,16 +1,135 @@
+                       What is Lorem Ipsum?
+                   </h1>
+                   <p>
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy t
+                        ext ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived 
+                        not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s 
+                        with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus 
+                        PageMaker including versions of Lorem Ipsum.
+                        <br>
+                        <strong><i>Why do we use it?</i></strong>
+                        It is a long established fact that a reader will be distracted by the readable 
+                        content of a page when looking at its layout. The point of using Lorem Ipsum is 
+                        that it has a more-or-less normal distribution of letters, as opposed to using 
+                        'Content here, content here', making it look like readable English. Many desktop 
+                        publishing packages and web page editors now use Lorem Ipsum as their default 
+                        model text, and a search for 'lorem ipsum' will uncover many web sites still in 
+                        their infancy. Various versions have evolved over the years, sometimes by accident, 
+                        sometimes on purpose (injected humour and the like).
+                   </p>
+               </div>
+                <div class="col-sm-8">
+                   <img src="images/cake1.jpg" alt="" 
+                      class="img-fluid img-thumbnail" id="images"> 
+
+                   <div class="row" id="pic-div">
+                        <div class="col-sm-3">
+                            <img src="images/cake2.jpg" alt="" class="img-fluid img-thumbnail" id="images">
+                        </div>
+                        <div class="col-sm-3">
+                            <img src="images/cake3.jpg" alt="" class="img-fluid img-thumbnail" id="images">
+                        </div>
+                        <div class="col-sm-3">
+                            <img src="images/cake4.jpg" alt="" class="img-fluid img-thumbnail" id="images">
+                        </div>
+                        <div class="col-sm-3">
+                            <img src="images/cake5.jpg" alt="" class="img-fluid img-thumbnail" id="images">
+                        </div>
+                    </div>
+
+              </div>
+
+           </div>
+           
+           <div class="row">
+                <div class="col-sm-3">
+                    <img src="images/cake6.jpg" alt="" class="img-fluid img-thumbnail" id="images">
```

Comment:
You have duplicate ids with the name 'images'. This defeats the object of ids as they should be unique. The browser will actually highlight this issue now and can affect your site's google rankings I believe.